entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/tx/TransactionHelper.java

```diff
@@ -36,6 +40,17 @@
     private static final Logger LOG = Logger.getLogger(TransactionHelper.class.getName());
     private static volatile TransactionAccessFactory factory;
 
+    private static final String HIBERNATE_IMPLEMENTATION_VERSION;
+
+    static {
+        try {
+            HIBERNATE_IMPLEMENTATION_VERSION = Class.forName("org.hibernate.Session")
+                        .getPackage().getImplementationVersion();
+        } catch (ClassNotFoundException e) {
+            throw new IllegalArgumentException("Unsupported jpa provider!", e);
```

Comment:
This causes https://travis-ci.com/github/Blazebit/blaze-persistence/jobs/326000071#L3448

IMO this should not cause an exception

Reply:
This is redundant anyway.